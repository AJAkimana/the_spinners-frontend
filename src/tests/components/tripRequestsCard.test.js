/* eslint-disable no-unused-vars */
/* eslint-disable prettier/prettier */
/* eslint-disable no-undef */
import React from 'react';
import { mount, shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import TripInfoCard from '../../components/requests/tripRequestsCard';

describe('Test On ProfilePicture Component', () => {
    const wrapper = shallow(<TripInfoCard />);
    const component = wrapper.dive();

    describe('render()', () => {
        test('Should Render ProfilePicture Component', () => {
            expect(toJson(component)).toMatchSnapshot();
        });
    });
});