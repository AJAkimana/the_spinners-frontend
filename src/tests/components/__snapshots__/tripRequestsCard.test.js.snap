// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test On ProfilePicture Component render() Should Render ProfilePicture Component 1`] = `
<ForwardRef(Card)
  className="makeStyles-root-1"
  classes={
    Object {
      "root": "MuiCard-root",
    }
  }
>
  <WithStyles(ForwardRef(Typography))
    style={
      Object {
        "backgroundColor": "#0074D9",
        "color": "#ffff",
        "fontWeight": "bold",
        "paddingBottom": "10px",
        "paddingTop": "10px",
        "textAlign": "center",
      }
    }
  />
  <div
    style={
      Object {
        "marginLeft": "20px",
        "marginRight": "20px",
      }
    }
  >
    <p>
      <span
        style={
          Object {
            "fontWeight": "bold",
            "paddingRight": "5px",
          }
        }
      >
        Origin:
      </span>
      <span />
    </p>
    <p>
      <span
        style={
          Object {
            "fontWeight": "bold",
            "paddingRight": "5px",
          }
        }
      >
        Destination:
      </span>
      <span />
    </p>
    <p>
      <span
        style={
          Object {
            "fontWeight": "bold",
            "paddingRight": "5px",
          }
        }
      >
        Departure Date:
      </span>
      <span />
    </p>
    <p>
      <span
        style={
          Object {
            "fontWeight": "bold",
            "paddingRight": "5px",
          }
        }
      >
        Return Date:
      </span>
      <span />
    </p>
  </div>
  <WithStyles(ForwardRef(Divider)) />
  <p
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    <span
      style={
        Object {
          "color": "#C62828",
          "fontWeight": "bold",
        }
      }
    />
  </p>
</ForwardRef(Card)>
`;
